### ISSUE TITLE
Eliminate double rounding-errors

### ISSUE THREAD

**Issue by aquila12** [yorkhackspace/meArm#4]
Currently, output from the IK is in radians (as a float).  This is converted into degrees (as an integer), and later into microseconds (also as an integer).  The second conversion is done within the servo library itself, which also exposes a writeMicroseconds method.

Some rounding error could be eliminated by directly converting from radians into microseconds, and omitting the second conversion (and the intermediate, granular, degree measurement).

**Comment by lexbailey** [comment]
Would switching to this require a recalibration for a given working arm?

**Comment by aquila12** [comment]
No; well, sort of - for a given set of existing cal values, we could
convert them (and provide helpers for that).

It might be worth creating an example sketch for calibration - disabling
all servos to start with, and setting each in turn to set it up properly.
